packages/react-core/src/demos/Chart/PrometheusChart.md

```diff
@@ -0,0 +1,207 @@
+  Text
+} from '@patternfly/react-core';
+import {
+  Chart,
+  ChartArea,
+  ChartAxis,
+  ChartGroup,
+  ChartLegend,
+  ChartLegendTooltip,
+  ChartLine,
+  ChartStack,
+  ChartThemeColor,
+  ChartThemeVariant,
+  ChartVoronoiContainer,
+  createContainer,
+  getCustomTheme
+} from '@patternfly/react-charts';
+import { css } from '@patternfly/react-styles';
+import BellIcon from '@patternfly/react-icons/dist/js/icons/bell-icon';
+import CogIcon from '@patternfly/react-icons/dist/js/icons/cog-icon';
+import HelpIcon from '@patternfly/react-icons/dist/js/icons/help-icon';
+import imgBrand from '@patternfly/react-core/src/components/Brand/examples/pfLogo.svg';
+import imgAvatar from '@patternfly/react-core/src/components/Avatar/examples/avatarImg.svg';
+import mockData from './mockData.js';
+
+PrometheusChart = () => {
+  const [activeItem, setActiveItem] = React.useState(0);
+  const [width, setWidth] = React.useState(0);
+  const [isStacked, setIsStacked] = React.useState(true);
+  const containerRef = React.useRef(null);
+  const legendData = [];
+  const tooltipSeriesNames = [];
+  const results = mockData.data.result;
+
+  const handleResize = () => {
+    if (containerRef.current && containerRef.current.clientWidth) {
+      setWidth(containerRef.current.clientWidth);
+    }
+  };
+
+  React.useEffect(() => {
+    handleResize();
+    window.addEventListener('resize', handleResize);
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
+
+  const timeFormatter = new Intl.DateTimeFormat('en', {
+    hour: 'numeric',
+    minute: 'numeric'
+  });
+
+  const dateTimeFormatterWithSeconds = new Intl.DateTimeFormat('en', {
+    month: 'short',
+    day: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+    second: 'numeric',
+    year: 'numeric'
+  });
+
+  const formatSeriesValues = values => {
+    const newValues = values.map(v => {
+      const y = Number(v[1]);
+      return {
+        x: new Date(v[0] * 1000),
+        y: Number.isNaN(y) ? null : y
+      };
+    });
+    return newValues;
+  };
+
+  const data = results.map(({ metric, values }) => {
+    return [metric, formatSeriesValues(values)];
+  });
+
+  data.forEach(([metric, values], i) => {
+    const name = `cpu${metric.cpu}`;
+    legendData.push({ name, childName: metric.cpu });
+    tooltipSeriesNames.push(name);
+  });
+
+  const CursorVoronoiContainer = createContainer('voronoi', 'cursor');
+
+  const graphContainer = (
+    <CursorVoronoiContainer
+      cursorDimension="x"
+      labels={({ datum }) => `${datum.y !== null ? datum.y : 'no data'}`}
+      labelComponent={
+        <ChartLegendTooltip legendData={legendData} title={datum => dateTimeFormatterWithSeconds.format(datum.x)} />
+      }
+      mouseFollowTooltips
+      voronoiDimension="x"
+      voronoiPadding={50}
+    />
+  );
+
+  const theme = getCustomTheme(ChartThemeColor.multiUnordered, ChartThemeVariant.light);
+  const colors = theme.line.colorScale;
```

Comment:
Unless you are changing the theme, you don't need this. You can simply define the chart's color scale using:

`themeColor={ChartThemeColor.multiUnordered}`